core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCRealmConstants.java

```diff
@@ -552,7 +552,6 @@ public final class JDBCRealmConstants {
     public static final String REMOVE_ABANDONED_TIMEOUT = "removeAbandonedTimeout";
     public static final String LOG_ABANDONED = "logAbandoned";
     public static final String CONNECTION_PROPERTIES = "connectionProperties";
-    public static final String INIT_SQL = "initSQL";
```

Comment:
Shall we keep the public constant?

Reply:
resolved in https://github.com/wso2/carbon-kernel/pull/3982/commits/a0236d13b6fa401654fd6a52040a914e7ba3acb3